src/debugProtocol/Debugger.ts

```diff
@@ -257,6 +267,17 @@ export class Debugger {
         }
     }
 
+    public async executeCommand(sourceCode: string, stackFrameIndex: number = this.stackFrameIndex, threadIndex: number = this.primaryThread) {
```

Comment:
Make this function generic so external callers can have more control over the response type.
```suggestion
    public async executeCommand<Response extends ExecuteResponseV3>(sourceCode: string, stackFrameIndex: number = this.stackFrameIndex, threadIndex: number = this.primaryThread) {
```